code/game/objects/effects/effect_system.dm

```diff
@@ -548,35 +553,6 @@ steam.start() -- spawns the effect
 					if(smoke) smoke.delete()
 					src.total_smoke--
 
-// Goon compat.
-/datum/effect/effect/system/chem_smoke_spread/fart

```

Comment:
it's originally what having "toxic farts" generated--eventually N3X15 fixed that though when I informed him how it really worked on Goon....that said, the code stayed.